m4_sessions/1705-inning/UBI.md

```diff
@@ -18,7 +15,13 @@ Our mission is to “unlock human potential.” The software we write enables it
 solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs
 they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
 
-
+## Outline
+1. Into/ General UBI info - 10 min
+2. General UBI info - Video 10min
+3. Individual Reflection on pros and cons - 5 min
+3. Break into small groups and discuss pros and cons - 15 min
```

Comment:
How large are groups?  3-4?  How can you break them up?  Stand up and count off?  Or could you put them in groups when they come in based off their opinions on UBI (line up on strongly agree to strongly disagree and then mix them?)